viewing_party/party.py

```diff
@@ -1,23 +1,128 @@
 # ------------- WAVE 1 --------------------
 
+from multiprocessing.sharedctypes import Value
+from re import I
+
+from tests.test_constants import GENRE_1, MOVIE_TITLE_1, RATING_1

```

Comment:
I don't see these packages used anywhere, so we can get rid of them. 

I see you used some test data to play around with! Good idea. Now that you are done and have submitted your final submission, we should always get rid of any testing, comments, or debugging statements that aren't part of the final code

```suggestion

```